mw-webapp/src/component/header/Header.tsx

```diff
@@ -19,6 +19,33 @@ const LOGO_TEXT = "master's way";
 export const Header = () => {
   const [user, setUser] = useState<User | null>(null);
 
+  const navigationLinks = [
```

Comment:
move const in separate file

Reply:
I can't do this beacause some properties of the navigationLinks depend on user